linera-client/src/client_options.rs

```diff
@@ -258,6 +258,10 @@ pub enum ClientCommand {
 
         /// Amount to transfer
         amount: Amount,
+
+        /// User Data
```

Comment:
Let's add to the documentation that it can only be up to 32 bytes.

Reply:
Pls check this documentation. https://github.com/linera-io/linera-protocol/pull/2369/commits/0a180e5b80489aa4cf3a945be60da2decb315743